docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -13,154 +13,220 @@ isTocNested: true
+  WITH (replica_placement='{"num_replicas": 3, "placement_blocks": [
+    {"cloud":"aws","region":"us-east-1","zone":"us-east-1a","min_num_replicas":1},
+    {"cloud":"aws","region":"us-east-1","zone":"us-east-1b","min_num_replicas":1},
+    {"cloud":"aws","region":"us-east-1","zone":"us-east-1c","min_num_replicas":1}]}');
+
+CREATE TABLE multi_zone_table (id INTEGER, field text)
+  TABLESPACE us_east_region_tablespace SPLIT INTO 1 TABLETS;
 ```
 
+Now let us measure the latencies incurred for INSERTs and SELECTs on this table, where the client is in us-east-1a zone:
+
 ```sql
-CREATE INDEX employee_no_idx ON employees(employee_no) 
-TABLESPACE us_east_tablespace;
+yugabyte=# INSERT INTO multi_zone_table VALUES (1, 'field1'), (2, 'field2'), (3, 'field3');
 ```
 
-The preceding statements ensure that data in the `employees` and `employee_no_idx` tables is present based on the placement policies specified for `us_west_tablespace` and `us_east_tablespace` respectively.
+```output
+Time: 11.825 ms
+```
 
-By default, indexes are placed according to the cluster configuration.
+```sql
+yugabyte=# SELECT * FROM multi_zone_table;
+```
 
-### How to Use a Single-Zone Tablespace
+```output
+ id | field
+----+--------
+  1 | field1
+  3 | field3
+  2 | field2
+(3 rows)
 
-There is no difference between creating tables and indexes that use multi-zone tablespaces and those that use single-zone tablespaces. 
+Time: 4.145 ms
+```
 
-Using the single-zone tablespace defined in [How to Create a Single-Zone Tablespace](how-to-create-a-single-zone-tablespace), you can apply the YSQL's `TABLESPACE` option to `CREATE TABLE` and `CREATE INDEX` statements, as follows:
+## Create a multi-region table
 
-```sql
-CREATE TABLE employees (
-  employee_no integer PRIMARY KEY,
-  name text,
-  department text,
-  change_date date
-)
-TABLESPACE us_west_1a_tablespace;
-```
+Now let us create a table that is spread across multiple regions as shown in the following diagram:
+
+![Multi Region Table](/images/explore/tablespaces/multi_region_table.png)
 
 ```sql
-CREATE INDEX employee_no_idx ON employees(employee_no) 
-TABLESPACE us_east_1a_tablespace;
+CREATE TABLESPACE us_multi_region_tablespace
+  WITH (replica_placement='{"num_replicas": 3, "placement_blocks": [
+    {"cloud":"aws","region":"us-east-1","zone":"us-east-1b","min_num_replicas":1},
+    {"cloud":"aws","region":"ap-south-1","zone":"ap-south-1a","min_num_replicas":1},
+    {"cloud":"aws","region":"eu-west-2","zone":"eu-west-2c","min_num_replicas":1}]}');
+
+CREATE TABLE multi_region_table (id INTEGER, field text)
+  TABLESPACE us_multi_region_tablespace SPLIT INTO 1 TABLETS;
 ```
 
-The preceding statements ensure that data in the `employees` and `employee_no_idx` tables is present based on the placement policies specified for `us_west_1a_tablespace` and `us_east_1a_tablespace` respectively.
+Now let us measure the latencies incurred for INSERTs and SELECTs on this table, where the client is in us-east-1a zone:
```

Comment:
The following demonstrates how to measure the latencies incurred for INSERTs and SELECTs on this table, where the client is in us-east-1a zone: